docs/localization/LOCALIZATION.md

```diff
@@ -690,18 +668,14 @@ For example if your `SimpleHeaderPresComponent` has `o3r-simple-header-pres` sel
 
 ```
 
-
-
 # Intelligent fallback support
 
 The fallback hierarchy has been added to the localization service which explained in detail below. As the last option, the default fallback language picked as usual.
 
-
-## Scenario 1: Fallback based on **fallbackLocalesMap**, **supportedLocales** language code.
+## Scenario 1: Fallback based on **fallbackLocalesMap**, **supportedLocales** language code
 
 In case **fallbackLocalesMap** is provided and the targeted translation language is unavailable in supported locales list, The priority goes to **fallbackLocalesMap**, to see if targeted translation language can be mapped with the fallback map configured. The second priority goes to **supportedLocales**, to map targeted translation language to the first closest possible language, the locale can be different. If none match, it will fall back to the **default language**.
```

Comment:
```suggestion
In case **fallbackLocalesMap** is provided and the targeted translation language is unavailable in supported locales list, the priority goes to **fallbackLocalesMap**, to see if the targeted translation language can be mapped to the fallback map. Then, the second priority goes to **supportedLocales**, to map the targeted translation language to the first closest possible language -- for instance a different locale of the requested language (en-GB to en-US). 

If none matches, it will fallback to the **default language**.
```